docs/usage/configuration-options.md

```diff
@@ -1102,6 +1102,19 @@ If you allow Renovate to run on "Selected" repositories, `forkProcessing` will b
 Only the `onboardingConfigFileName` (which defaults to `renovate.json`) is supported for `forkProcessing`.
 You can't use other filenames because Renovate only checks the default filename when using the Git-hosting platform's API.
 
+## gerritLabelMapping
+
+The name of the Gerrit labels to use for the `StabilityDays` and `Merge-Confidence` status.
+
+```json
+{
+  "stabilityDaysLabel": "Renovate-Stability",
+  "mergeConfidenceLabel": "Renovate-Confidence"
+}
+```
+
+If one or both are unset, then these information will not be passed to the Gerrit changes.
```

Comment:
```suggestion
You must set _both_ labels, for Renovate to pass the information to the Gerrit changes.
```

I think this is easier to understand, check if it's technically correct. 😉

Reply:
no,both features can be used independently.

I'm still unsure if the `minimumReleaseAge` is really useful, because if the release-age is not reached then no PR will be created and therefore no branch-status will be set.

The status can't have any other value then green or?

Reply:
> no,both features can be used independently.

How about this then:

```suggestion
You must set a label, for Renovate to pass the information to the Gerrit changes.
```

> I'm still unsure if the `minimumReleaseAge` is really useful, because if the release-age is not reached then no PR will be created and therefore no branch-status will be set.

I think your PR will allow Gerrit-users to set labels for Renovate's PRs? I don't see how setting a `minimumReleaseAge` label is helpful in that case. If the `minimumReleaseAge` is _not_ met, Renovate won't create the PR in the first place. So in effect users would add a `minimumReleaseAge` label that tells them nothing special?

> The status can't have any other value then green or?

I don't know how `renovate/stability-days` works internally. I have seen this behavior myself:

1. Use Dependency Dashboard to force Renovate to create a PR that's still watiting for the `minimumReleaseAge` days to be met
2. Renovate creates PR
3. Look at `renovate/stability-days`, that's _not_ met, see yellow check box
4. Once `minimumReleaseAge` is reached, check turns green

So it looks like there are at least two states, yellow and green. There may be a third red state though, or other states. 😄

Reply:
Okay, but with Gerrit there is no Dependency-Dashboard to force the creation of PRs.
Then this label seems to be useless for Gerrit. I think i should remove this label.

Is it possible to test the Merge-Confidence Feature? 
Last time I've tried it looks like i need a special subscription (Token?) for this feature?

Reply:
> > Okay, but with Gerrit there is no Dependency-Dashboard to force the creation of PRs. Then this label seems to be useless for Gerrit. I think i should remove this label.
> 
> No, It depends on `internalChecksFilter`. https://docs.renovatebot.com/configuration-options/#minimumreleaseage
> 

Okay, good to know. But i've already removed the Gerrit-Support for this label.

And I guess that setting `internalChecksFilter=none` will probably only affect PR creation but not the creation of branches?

And with Gerrit each branch creation will automatically create a PR. (implicite)

Reply:
```suggestion
You must set at least _one_ of the labels above for Renovate to pass the information to the Gerrit changes.
```

I think the only thing left to do here is to reword the sentence?

Let's put our points positively.